tests/test_client.py

```diff
@@ -1,16 +1,25 @@
+import unittest
+from unittest import mock
+
 import pytest
+import requests
+import stamina
 
 import predicthq
 from predicthq import endpoints
 from predicthq.endpoints.oauth2.schemas import AccessToken
-from predicthq.exceptions import ClientError, ServerError
-from tests import with_mock_responses, with_config, with_mock_client, load_fixture
+from predicthq.exceptions import ClientError, RateLimitError, ServerError
+from tests import load_fixture, with_config, with_mock_client, with_mock_responses
 
 
-class ClientTest(object):
+class ClientTest(unittest.TestCase):
```

Comment:
This test class never ran before because it had to inherit from `unittest.TestCase` for pytest to pick it up otherwise pytest was silently skipping it.

Reply:
😰 That's been around for a while... Good catch Medhat !

Reply:
Terrifying - nice catch @medhatphq